database/migrations/2024_01_03_000000_add_standalone_system.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\App;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        if (App::environment('testing')) {
+            return;
+        }
```

Comment:
I must confess I'm really not in love with this.

However, there are numerous tests that rely on only a certain number of console IDs existing in the test database.

Part of me still feels like it'd be better if this write operation were not in a migration.

Reply:
I think we should just craft an insert statement and run it directly against the prod DB.

As long as the seeder populates it for general use, any tests relying on this specific console can add it themselves.